apps/web/src/app/host-dashboard/page.tsx

```
@@ -0,0 +1,1131 @@
+                      onClick={() => !isPast && toggleDateSelection(date)}
+                      disabled={isPast}
+                      className={`p-2 text-sm rounded-lg transition-colors ${
+                        isPast 
+                          ? 'text-gray-300 cursor-not-allowed'
+                          : isSelected
+                          ? 'bg-blue-600 text-white'
+                          : isCurrentMonth
+                          ? 'text-gray-900 hover:bg-gray-100'
+                          : 'text-gray-400 hover:bg-gray-50'
+                      }`}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)